crates/fluvio-controlplane-metadata/Cargo.toml

```diff
@@ -27,6 +27,7 @@ flate2 = { version = "1.0.22", optional = true }
 semver = "1.0"
 toml = { version = "0.5.9", optional = true }
 lenient_semver = "0.4.2"
+wasmparser = "0.93.0"
```

Comment:
The current implementation follows the associated function pattern so the validation can be used in other use cases. This opens the possibility of having this dependency as a feature as well.